modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/rest-openapi.yaml

```diff
@@ -1214,6 +1218,13 @@ components:
                 catalogId:
                     format: int64
                     type: integer
+                customFields:
+                    items:
+                        $ref: "#/components/schemas/CustomField"
+                    type: array
+                defineExternally:
+                    example: true
```

Comment:
Do we have any other examples of something that is named like this in .yaml?

Reply:
defineExternally specifically no, but there's similar ones like facetable, required, allowBackOrder, neverExpire, which are basically the name of the toggle option in the UI, so that's why i put defineExternally. But i guess it could also be definedExternally

Reply:
Yes, it should be `definedExternally`. That's how it's stored and identified as.

Reply:
Once you have done the change, you can send it to liferay-commerce. Make sure you only run ci:test:sf